FlowCrypt/src/androidTest/java/com/flowcrypt/email/SendMsgTest.kt

```diff
@@ -404,17 +386,8 @@ class SendMsgTest {
 
     processOutgoingMessageInfo(outgoingMessageInfo)
 
-    val forwardedAttachmentsDownloaderWorker =
-      TestListenableWorkerBuilder<ForwardedAttachmentsDownloaderWorker>(context).build()
-    val messagesSenderWorker = TestListenableWorkerBuilder<MessagesSenderWorker>(context).build()
     runBlocking {
-      val forwardedAttachmentsDownloaderWorkerResult = forwardedAttachmentsDownloaderWorker.doWork()
-      assertThat(
-        forwardedAttachmentsDownloaderWorkerResult,
-        `is`(ListenableWorker.Result.success())
-      )
-      val messagesSenderWorkerResult = messagesSenderWorker.doWork()
-      assertThat(messagesSenderWorkerResult, `is`(ListenableWorker.Result.success()))
+      delay(5000)
```

Comment:
unfortunately, yes. it needs some time to wait while all workers will complete their job.

Before, I use `messagesSenderWorker.doWork()`. But it doesn't work well here. Because under the hood one worker calls another one. That's why I've removed that code and replaced it with the delay.

Reply:
Well, static delay doesn't seem to statble.
Maybe we can file for another issue?

Reply:
It should be stable. If not, I will have to change this test at all. Only time will show it. unfortunately, this test is testing multithreading things. That's why for simplicity I use only delay here and didn't spend a few days to change this test.

Reply:
Ok, then